packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -590,6 +595,9 @@ export class IndexedDbPersistence implements Persistence {
           }
 
           if (!this.isLocalClient(currentPrimary)) {
+            if (this.force) {
+              return true;
+            }
```

Comment:
I'm not sure this is the right place to do this.  We only want to respect `force` on initial client startup.  Else, if you launch two tabs with `force`: true, I think they could both end up broken because they both steal the lease from the other one or something (it's kind of race-y so I'm not 100% sure if it would happen in practice).

So I think maybe we want to add a `force` _parameter_ to updateClientMetadataAndTryBecomePrimary() and then if it's true we can just skip the canActAsPrimary() check.  And we will pass true only from IndexedDbPersistence.start() if settings.force == true.